### ISSUE TITLE
August runs tables are missing data

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/httparchive#129]
The desktop/mobile runs.requests tables are missing for the 8/1 crawl.

**Comment by rviscomi** [comment]
Error in the sync_csv script:

```
Submitting new requests import runs.2017_08_01_requests to BigQuery: requests_ag.gz
Waiting on bqjob_r1b4b1ac50a676199_0000015e0b7b2b24_1 ... (0s) Current status: DONE   
BigQuery error in load operation: Error processing job
'httparchive:bqjob_r1b4b1ac50a676199_0000015e0b7b2b24_1': CSV table encountered too many errors, giving
up. Rows: 2447; errors: 1.
Failure details:
- gs://httparchive/Aug_1_2017/requests_ag.gz: CSV table references
column position 23, but line starting at position:2375002 contains
only 23 columns.
```

There was another issue (I'll cross reference when I find it) that had CSV issues. The fix there was to convert newlines into commas so maybe it's adding a comma where it shouldn't and the parser is getting confused.

Would be great to move off CSV "Once And For All" :tm: and avoid this kind of thing forever. For now, my next move will be to bisect the CSV until I find which line is causing the error. It's interesting that whatever it is is only affecting the requests tables and it's both desktop/mobile.

**Comment by rviscomi** [comment]
8/1 tables are back to normal. I'm currently rerunning the 8/15 scripts, so updating this issue to track both.

**Comment by rviscomi** [comment]
8/15 LGTM.
